packages/mdc-top-app-bar/short/foundation.ts

```diff
@@ -45,33 +47,61 @@ export class MDCShortTopAppBarFoundation extends MDCTopAppBarBaseFoundation {
       this.adapter_.addClass(cssClasses.SHORT_HAS_ACTION_ITEM_CLASS);
     }
 
-    // this is intended as the short variant must calculate if the
-    // page starts off from the top of the page.
-    this.handleTargetScroll();
+    // If initialized with SHORT_COLLAPSED_CLASS, the bar should always be collapsed
+    this.setAlwaysCollapsed(
+      this.adapter_.hasClass(cssClasses.SHORT_COLLAPSED_CLASS));
+  }
+
+  /**
+   * Set if the short top app bar should always be collapsed.
+   *
+   * When set to `true`, the bar collapses.
+   * When set to `false`, the bar will determine if it should collapse based on scroll position.
+   */
+  setAlwaysCollapsed(value: boolean) {
+    this.isAlwaysCollapsed_ = !!value;
+    if (this.isAlwaysCollapsed_) {
+      this.collapse_();
+    } else {
+      // let handleTargetScroll determine if the bar should be collapsed
+      this.handleTargetScroll();
+    }
+  }
+
+  getAlwaysCollapsed() {
+    return this.isAlwaysCollapsed_;
   }
 
   /**
    * Scroll handler for applying/removing the collapsed modifier class on the short top app bar.
    * @override
    */
   handleTargetScroll() {
-    if (this.adapter_.hasClass(cssClasses.SHORT_COLLAPSED_CLASS)) {
+    if (this.isAlwaysCollapsed_) {
       return;
     }
     const currentScroll = this.adapter_.getViewportScrollY();
 
     if (currentScroll <= 0) {
       if (this.isCollapsed_) {
-        this.adapter_.removeClass(cssClasses.SHORT_COLLAPSED_CLASS);
-        this.isCollapsed_ = false;
+        this.uncollapse_();
       }
     } else {
       if (!this.isCollapsed_) {
-        this.adapter_.addClass(cssClasses.SHORT_COLLAPSED_CLASS);
-        this.isCollapsed_ = true;
+        this.collapse_();
       }
     }
   }
+
+  private uncollapse_() {
```

Comment:
Did you test this in screenshot demo: test/screenshot/spec/mdc-top-app-bar/classes/baseline-short.html ?

Reply:
Yep: https://storage.googleapis.com/mdc-web-screenshot-tests/dfreedm/2019/08/19/23_45_33_613/report/report.html?utm_source=cli&utm_medium=test_results